storybook/src/components/Field/Field.stories.tsx

```diff
@@ -23,7 +23,7 @@ export const Default: Story = {
   render: (args) => (
     <Field invalid={args.invalid}>
       <Label htmlFor="input1">Waar gaat het om?</Label>
-      <TextInput id="input1" aria-invalid={args.invalid ? true : undefined} />
+      <TextInput id="input1" invalid={args.invalid} />
```

Comment:
So, again, it's either `true` or `undefined`. Then I'd say replace the `boolean` type with `true` everywhere and always make it optional.